src/test/java/uk/gov/ons/census/casesvc/service/CaseServiceTest.java

```diff
@@ -197,7 +196,7 @@ public void testCreateCCSCaseWithRefusalNotReceived() {
             });
 
     // When
-    Case actualCase = underTest.createCCSCase(caseId, sampleUnit, null, false);
+    Case actualCase = underTest.createCCSCase(caseId, sampleUnit);
```

Comment:
If we're no longer dealing with refusals in CCS case events then I think this test can go?

Reply:
One of the tests has now been renamed to not reference refusals, the other one has been deleted